jsbits/Scenario.js

```diff
@@ -115,10 +142,11 @@ function gm$updateProps(bArray, values) {
  * @param fc - FeatureCollection
  * @returns {[points:Feature,lines:Feature,surfaces:Feature,deletes:Number(geomID),errors:string,cmin,cmax,dims]}
  */
-function gm$smartProcessFeatureCollection(fc, defVec, maxGeomId) {
+
+function gm$smartProcessFeatureCollection(fc, coorSys, defVec, maxGeomId) {
```

Comment:
I think there are 3 possible conditions in this case:
1. Transform WGS'84 to Metric (because no geoLocation provided).
2. No conversion (because geoLocation provided).
3. There is no root information, the input is FeatureCollection, so we need to do heuristic.